feature/pomodoro/build.gradle.kts

```diff
@@ -1,10 +1,13 @@
 plugins {
     id("stopwatch.lint")
     id("stopwatch.android.feature")
+    alias(libs.plugins.org.jetbrains.kotlin.android)
 }
 
 android {
     namespace = "com.timers.stopwatch.feature.pomodoro"
 }
 dependencies {
+    implementation(project(mapOf("path" to ":core:database")))
```

Comment:
feature modules should not implement :core:database and :core:data, as per clean architecture feature are supposed to use only the domain layer (repository or/and use cases), not the implementation (data, database)